graspy/match/gmp.py

```diff
@@ -290,11 +284,29 @@ def f(x):  # computing the original optimization function
             score_new = np.trace(
                 np.transpose(A) @ B[np.ix_(perm_inds_new, perm_inds_new)]
             )  # computing objective function value
-
-            if obj_func_scalar * score_new < obj_func_scalar * score:  # minimizing
-                score = score_new
-                perm_inds = np.zeros(n, dtype=int)
-                perm_inds[permutation_A] = permutation_B[perm_inds_new]
+            return score_new, perm_inds_new
+            # end of the _fit_single_init function
+
+        par = Parallel(n_jobs=-1)
+        score_new, perm_inds_new = map(
```

Comment:
this is too much for one line. can you have `outs=par(...)` on a separate line or something like that